secure/storage/src/storage.rs

```diff
@@ -2,11 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::{Error, Policy, Value};
+use libra_crypto::ed25519::{Ed25519PrivateKey, Ed25519PublicKey};
+use libra_crypto::{PrivateKey, Uniform};
```

Comment:
nit use a single use statement

Reply:
Done, thanks! (I also saw Zekun's comment about this, so I might consider using the nightly build to automatically fix these types of problems)..